python/pyarrow/tests/test_array.py

```diff
@@ -3579,6 +3579,47 @@ def test_run_end_encoded_from_buffers():
                                            1, offset, children)
 
 
+def test_run_end_encoded_from_array_with_type():
+    run_ends = [1, 3, 6]
+    values = [1, 2, 3]
+    ree_type = pa.run_end_encoded(pa.int32(), pa.int64())
+    expected = pa.RunEndEncodedArray.from_arrays(run_ends, values,
+                                                 ree_type)
+
+    arr = [1, 2, 2, 3, 3, 3]
+    result = pa.array(arr)
+    assert result.equals(expected)
```

Comment:
```suggestion
```

This part is failing because to `type` is specified

Reply:
It's a bit annoying I can't test locally, it's making me quite confused. Looking at this part now, I do not know what I was trying to test really. Hope I will fix pytest soon =)